#wrench_topic    : "filtered_wrench_base"
wrench_topic    : "filtered_wrench_base"
dwrench_topic   : "delta_force"
vel_topic       : "current_velocity"
pos_topic       : "current_pose"

assistance_topic: "assistance"
reference_traj  : "human_target"

robot_has_human_reference: false

#K_assistance: 0.0002
#K_assistance: 0.0001
#K_assistance: 0.0001

# UR5 
#K_tras: 0.0001
#K_rot: 0.000001

#UR10
K_tras: 0.00005
K_rot: 0.0000001
